sql/programlabels.sql

```diff
@@ -3,5 +3,7 @@ update program_attribute_type set description = 'PROGRAM_MANAGEMENT_PATIENT_STAG
 update program_attribute_type set description = 'PROGRAM_MANAGEMENT_PATIENT_TREATMENT_DATE', name = 'PROGRAM_MANAGEMENT_PATIENT_TREATMENT_DATE' where name = 'Date';
 update program_attribute_type set description = 'PROGRAM_MANAGEMENT_PATIENT_COMORBIDITES', name = 'PROGRAM_MANAGEMENT_PATIENT_COMORBIDITES' where name = 'Co-morbidites';
 update program_attribute_type set description = 'PROGRAM_MANAGEMENT_PATIENT_DOCTOR_IN_CHARGE', name = 'PROGRAM_MANAGEMENT_PATIENT_DOCTOR_IN_CHARGE' where name = 'Doctor';
+insert into program_attribute_type(name, description, datatype,creator, date_created, uuid) values("PROGRAM_MANAGEMENT_LABEL_THERAPEUTIC_LINE","PROGRAM_MANAGEMENT_LABEL_THERAPEUTIC_LINE","org.bahmni.module.bahmnicore.customdatatype.datatype.CodedConceptDatatype", 1, NOW(), "a8bc4608-eaae-4610-a842-d83d6261ea49");
+update program_attribute_type set datatype_config=(select concept_id from concept where uuid="a8bc4608-eaae-4610-a842-d83d6261ea49") where uuid="a8bc4608-eaae-4610-a842-d83d6261ea49";
```

Comment:
@IipperMangesh The existing concept set Therapeutic line with uuid a8bc4608-eaae-4610-a842-d83d6261ea49, has 

1st line (3828)
2nd line (3829)
3rd line (3830)
1st substituted line (3831)
2nd substituted line (3832)
3rd substituted line (3833)

as members. This story requires only

1st line (3828)
2nd line (3829)
3rd line (3830)

as members. So please create a new concept.

Reply:
You can create the new concept in 1020_hiv_program_attribute_types.csv file and call it "HIV Therapeutic line" and "VIH  Ligne thérapeutique"